module/VuFind/src/VuFind/ILS/Driver/SierraRest.php

```diff
@@ -1504,6 +1517,76 @@ protected function makeRequest(
         $method = 'GET',
         $patron = false,
         $returnStatus = false
+    ) {
+        // Status logging callback:
+        $statusCallback = function ($attempt, $exception) use (
+            $hierarchy,
+            $params,
+            $method
+        ): void {
+            $apiUrl = $this->config['Catalog']['host'];
```

Comment:
Would it make sense to have a getUrlFromHierarchy support method rather than having this logic in two places?

Reply:
Yes, method added.